soc/realtek/ec/rts5912/device_power.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+ * Author: Lin Yu-Cheng <lin_yu_cheng@realtek.com>
+ */
+
+#include <zephyr/logging/log.h>
+#include <zephyr/kernel.h>
+#include <zephyr/pm/pm.h>
```

Comment:
why is PM involved at all here?

Reply:
We remove the include of pm.h here but we keep including pm.h at power.c because we use the macro of PM_STATE_SUSPEND_TO_IDLE & PM_STATE_SUSPEND_TO_RAM here, thanks.